src/app/data-entries/select/select-header-footer/select-header-footer.component.scss

```diff
@@ -9,8 +9,15 @@
     margin-left: 48px;
 }
 
-.igx-drop-down__item {
-    justify-content: space-between;
+:host ::ng-deep {
+    .igx-drop-down__item {
```

Comment:
Do not target inner classes. Add "user-specific" classes directly to the element or use the element selector instead. We are trying to teach people how to do things the right way. They shouldn't rely on the component classes we use, as they can change.